src/new-command/command-name-lambda.js

```diff
@@ -0,0 +1 @@
+exports.handler = async (event) => {}
```

Comment:
@ramyaragupathy 
I left this empty but I was wondering if I should include the code to parse the event payload and to extract the `response_url` and parameters?
```
  const snsMessage = JSON.parse(event.Records[0].Sns.Message)
  const responseURL = decodeURIComponent(snsMessage.response_url)
  const commandParameters = snsMessage.text
```

Other things I can think of including is the generic error block:
```
const ERROR_BLOCK = {
  response_type: 'ephemeral',
  text:
    ':x: Something went wrong with your request. Please try again and if the error persists, post a message at <#C319P09PB>',
}
```
and the `sendToSlack` helper function.
```
const sendToSlack = async (responseURL, message) => {
  await fetch(responseURL, {
    method: 'post',
    body: JSON.stringify(message),
    headers: {
      'Content-Type': 'application/json',
    },
  })
}
```

These might just cause more confusion though so I left it out.